test/mocks/carbonio-shell-ui.tsx

```diff
@@ -20,14 +21,21 @@ const FakeIntegration = (): JSX.Element => <div data-testid="fake-component" />;
 const mockedAccount = generateAccount();
 const mockedAccounts = [mockedAccount];
 const mockedSettings = generateSettings();
+const mockedAppContext = {
+	isMessageView: true,
```

Comment:
I don't think this should be in common. It is something strictly related to mails, isn't it?

Reply:
yes you are right, I am moving it to mails